content/develop/connect/insight/rdi-connector.md

```diff
@@ -0,0 +1,158 @@
+
+Choose from the following options:
+
+- **Download from server** - Download already-defined pipelines from your RDI configuration.
+- **Upload from file** - Upload YAML pipeline files from your local computer.
+- **Create new pipeline** - Use Redis Insight's built-in editors to create new pipelines either from scratch or using one of the built-in templates.
+
+Each of these menu options will be described in more detail in subsequent sections.
+
+There are also equivalent buttons at the top of the editor pane for the first two of these functions.
+
+<img src="../images/ri-rdi-pl-editor-minibuttons.png">
+
+If you'd rather start with an empty configuration, exit the dialog, which will leave you in the **Configuration file** editor where you can begin editing your configuration pipeline.
+
+### Download pipelines from your RDI configuration
+
+Click the **Download from server** button in the **Start with your pipeline** dialog to download previously defined configuration and transformation job pipelines from your RDI server. The downloaded pipelines will be displayed in the **Pipeline management** pane. The configuration pipeline file will be displayed in the center panel.
+
+<img src="../images/ri-rdi-pl-dl.png">
+
+### Upload pipelines from your local machine
+
+Click the **Upload from file** button in the **Start with your pipeline** dialog to upload your YAML files from your local machine. The files must be stored in a zip file that has the following structure.
+
+```
+├── config.yaml
+└── jobs
+    └── job1.yaml
+```
+
+The `config.yaml` file, your configuration pipeline YAML file, is required. The `jobs` directory can be empty, as job pipelines are not required, but the empty directory must exist in the zip file. Otherwise, the `jobs` folder might include one or more transformation job pipeline files.
+
+### Create a new configuration pipeline using the built-in editor
+
+Click the **Create new pipeline** button in the **Start with your pipeline** dialog to create new pipeline data using the editor in Redis Insight. After doing so, you'll enter the **Configuration file** editor and you'll see a **Select a template** dialog in the upper right-hand corner of the editor.
+
+<img src="../images/ri-rdi-pl-editor-dlg.png">
+
+Make your selections in the provided fields:
+
+- **Pipeline type** is set to **Ingest**. This is the only available option.
+- **Database type** has six options:
+  - mongodb
+  - cassandra
+  - mysql
+  - oracle
+  - postgresql
+  - sqlserver
+
+After you make your selections and click **Apply**, Redis Insight will populate the editor window with an appropriate template. To start from scratch, click **Cancel**.
+
+<img src="../images/ri-rdi-pl-editor-template.png">
+
+### Create a new transformation job pipeline using the built-in editor
+
+In the **Pipeline Management** pane, click the `+` next to the **Jobs** folder and enter a name for the new job.
+Next, click the job name you just created.
```

Comment:
we need to mention that RDI uses a very specific YAML format and I think the reference to their documentation will be a good addition here - https://redis.io/docs/latest/integrate/redis-data-integration/ingest/data-pipelines/data-pipelines/#job-files

Reply:
also, it is worth mentioning that Redis Insight validates if the file has a correct YAML structure and highlights errors

It also indicates the files with undeployed changes (dots over the file names) so users can better understand what files they changed

Reply:
Done. Look for the next merge.